deploy/kubernetes/base/resources.yaml

```diff
@@ -42,7 +42,9 @@ rules:
   - apiGroups: [ "" ]
     resources: [ "endpoints" ]
     verbs: [ "get", "list", "watch", "create", "update", "patch" ]
-
+  - apiGroups: [ "apps" ],
+    resources: [ "daemonsets" ]
+    verbs: [ "get", "list", "watch", "create", "update", "patch" ],
```

Comment:
Do we need so many permissions? Or keeping `get` and `list` is just ok?

Reply:
done